profile

```diff
@@ -149,14 +149,15 @@ function update_profile() {
 
 function setup-role() {
   prompt AWS_PROFILE "What should we call this profile [no spaces]? (e.g. ops) " "ops"
-  prompt AWS_ACCOUNT_ID "What is your AWS Account ID? (e.g. 324149397721)" 
+  prompt AWS_ACCOUNT_ID "What is your AWS Account ID? (e.g. 324149397721)"
+  prompt AWS_ORGANIZATION_ACCOUNT_ID "What is your AWS Organization Account ID? (e.g. 324149397721)"
```

Comment:
mb more pretty will be this variant: `"What is your AWS Organization or Account ID? (e.g. 324149397721)"`

Reply:
We need both prompts.
See https://github.com/cloudposse/aws-assumed-role/pull/16#discussion_r148001767

Reply:
nvm, careless look